main.tf

```diff
@@ -39,8 +39,10 @@ data "aws_region" "current" {}
 data "aws_caller_identity" "current" {}
 
 locals {
-  region     = coalesce(var.region, data.aws_region.current.name)
-  account_id = data.aws_caller_identity.current.account_id
+  region                        = coalesce(var.region, data.aws_region.current.name)
+  account_id                    = data.aws_caller_identity.current.account_id
+  session_logging_kms_key_alias = var.session_logging_kms_key_alias
```

Comment:
Why assign this to a local if it's only being used once?

Reply:
I had previous include interpolation logic and handled that in the local block. Removing since there's string interpolation logic.